components/tools/rag-details.tsx

```
@@ -0,0 +1,159 @@
+                  <div className="flex flex-col gap-2">
+                    <div className="text-lg font-medium">Question</div>
+                    <p className="text-muted-foreground">{args?.question}</p>
+                  </div>
+                  <div className="flex flex-col gap-2">
+                    <div className="text-lg font-medium">Results</div>
+                    {isLoading && (
+                      <span className="text-muted-foreground">Getting results...</span>
+                    )}
+                    {result?.map((item: any, index: number) => (
+                      <div key={index} className="flex flex-col gap-2 relative">
+                        <div className="text-sm font-medium text-muted-foreground absolute bottom-0 right-0 m-[1px] text-[10px] bg-card rounded-br-xl rounded-t-md p-1">
```

ESLint

[error] Classnames text-sm, text-[10px] are conflicting!

(tailwindcss/no-contradicting-classname)